hack/verify-govet.sh

```diff
@@ -42,34 +42,5 @@ if [[ ${#targets[@]} -eq 0 ]]; then
   targets=$(go list ./... | egrep -v "/(third_party|vendor)/")
 fi
 
-# Do this in parallel; results in 5-10x speedup.
-pids=""
-for i in $targets
-  do
-  (
-    # Run go vet using goflags for each target specified.
-    #
-    # Remove any lines go vet or godep outputs with the exit status.
-    # Remove any lines godep outputs about the vendor experiment.
-    #
-    # If go vet fails (produces output), grep will succeed, but if go vet
-    # succeeds (produces no output) grep will fail. Then we just use
-    # PIPESTATUS[0] which is go's exit code.
-    #
-    # The intended result is that each incantation of this line returns
-    # either 0 (pass) or 1 (fail).
-    go vet "${goflags[@]:+${goflags[@]}}" "$i" 2>&1 \
-      | grep -v -E "exit status|GO15VENDOREXPERIMENT=" \
-      || fail=${PIPESTATUS[0]}
-    exit $fail
-  ) &
-  pids+=" $!"
-done
-
-# Count and return the number of failed files (non-zero is a failed vet run).
-failedfiles=0
-for p in $pids; do
-  wait $p || let "failedfiles+=1"
-done
-
-exit $failedfiles
+echo "${targets[@]}" \
+  | parallel -I {} go vet "${goflags[@]:+${goflags[@]}}" "{}" 2>&1
```

Comment:
it might be simpler just to do `| xargs go vet "${goflags[@]:+${goflags[@]}}"` here and let go take care of parallizing itself.

on my workstation, the argument list can be very large, so I saw no benefit from using `-P`. It might be helpful in other environments, possibly.